src/NSwag.AspNetCore/SwaggerUiSettingsBase.cs

```diff
@@ -35,7 +35,7 @@ public abstract class SwaggerUiSettingsBase : SwaggerSettings
         public SwaggerUiSettingsBase()
         {
             TransformToExternalPath = (internalUiRoute, request) =>
-                '/' + (request.GetBasePath()?.TrimEnd('/') + '/' + internalUiRoute?.TrimStart('/')).Trim('/');
+                ('/' + request.GetBasePath()?.TrimEnd('/') + '/' + internalUiRoute?.TrimStart('/')).Trim('/');
```

Comment:
I fixed that with the second commit. 

I think this should be fixed in the project because it was working in older versions. Upgrading to the latest version will cause errors for anyone who was using an empty path.